forest_forpriv/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('forpriv/intersection', views.forpriv_intersection, name='intersection'),
```

Comment:
```suggestion
    path('intersection', views.forpriv_intersection, name='intersection'),
```
We already know it's forpriv because we're in the app